controller_manager/src/controller_manager.cpp

```diff
@@ -183,9 +189,54 @@ ControllerManager::ControllerManager(
   {
     RCLCPP_WARN(get_logger(), "'update_rate' parameter not set, using default value.");
   }
+
+  if (!resource_manager->is_urdf_already_loaded())
```

Comment:
```suggestion
  if (!resource_manager_->is_urdf_already_loaded())
```
The resource_manager variable is already moved in the constructor, using the same might cause a segmentation fault

Reply:
I agree that `resource_manager_` should be used in this case.

Reply:
excellent catch!